docs/ingestion/native-batch-input-source.md

```diff
@@ -63,7 +63,7 @@ Sample specs:
       "type": "index_parallel",
```

Comment:
the documentation needs an update as well.

Reply:
Updated.